_maps/configs/minutemen_typhon.json

```
@@ -0,0 +1,46 @@
+	{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Typhon-class Light Exosuit Carrier",
+	"prefix": "CMSV",
+	"manufacturer": "Lanchester Shipyards",
+	"namelists": ["GENERAL", "CLIP MINUTEMEN", "NATURAL_AGGRESSIVE"],
+	"description": "The Typhon-Class is a robust conversion of an old miner-freighter, seeing employment by the Colonial Militia as a fast-response vessel against smaller pirate holdings, whilst it's spacious cargo decks enable relatively long-term deployment for it's crew size and the facilities needed to maintain an exosuit in working condition. The vessel commonly sees deployment in fringe sectors with a goal to combat pirate activity. The Typhon began it's service around the year 400 FSC. However, it's production only truly began after the Colonial Militia liberated Lanchester City. During the Frontiersmen War, the ship primarly acted as an escort for larger vessels.",
```

```suggestion
	"description": "The Typhon-class is a reliable conversion of an old miner-freighter, featuring spacious cargo decks that enable relatively long-term deployment for its crew size and the facilities needed to maintain an exosuit in working condition. The Typhon started service in 400 FS, serving as an escort for larger vessels just before the height of the Frontiersmen War, and later produced in large numbers after the liberation of Lanchester City. Nowadays, the nimble nature of Typhons make them well-suited to quickly respond to and deter small smuggling and pirate activity in fringe sectors.",
```

sorry if this is too heavy of a rewrite this started as correcting all the it's

---

All fine. Commited!